### PR TITLE
Authors page

### PR DESCRIPTION
**Pull request by a1exymoroz**
### What is this article about?
- Authors page
- Avatar component

### CODE REVIEW

File: src/app/core/layout/navigation/navigation.component.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export class NavigationComponent implements OnInit {
   protected readonly navigationItems = [
     { link: '/', label: 'Blog' },
     { link: '/principles', label: 'Principles' },
+    { link: '/authors', label: 'Authors' },
```
Comment (robinsonzimmermann) [inline src/app/core/layout/navigation/navigation.component.ts]: Let's not add it to the main navigation, please
```suggestion
```

Comment (a1exymoroz) [reply src/app/core/layout/navigation/navigation.component.ts]: I've already removed on my local branch🙂

File: src/app/core/layout/footer/footer.component.ts [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+    MatAnchor,
+    MatIcon,
+    RouterLink,
+  ],
   templateUrl: './footer.component.html',
   styleUrl: './footer.component.scss',
 })
-export class FooterComponent {}
+export class FooterComponent {
+  isAuthorsPage$ = this.router.events.pipe(
```
Comment (robinsonzimmermann) [inline src/app/core/layout/footer/footer.component.ts]: Please don't use the global footer for that, create a section in the author details page itself

File: src/app/components/button/button.component.scss [hunk 3]
```diff
@@ -8,7 +8,12 @@ blog-button {
...
-      --mdc-outlined-button-label-text-color
+        --mdc-outlined-button-label-text-color
     );
   }
 }
+
+footer {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (robinsonzimmermann) [inline src/app/components/button/button.component.scss]: This style isn't in the right scope

File: src/app/app.routes.ts [hunk 4]
```diff
@@ -42,6 +42,13 @@ export const routes: Routes = [
     loadComponent: () =>
       import('./features/post/post.component').then(m => m.PostComponent),
   },
+  {
+    path: 'authors',
```
Comment (robinsonzimmermann) [inline src/app/app.routes.ts]: Can we use `people`? This way we have both `people` and `people/:author` – see above

File: src/app/core/layout/footer/footer.component.ts (hunk 2 above)
Comment (a1exymoroz) [reply src/app/core/layout/footer/footer.component.ts]: Done

File: src/app/components/button/button.component.scss (hunk 3 above)
Comment (a1exymoroz) [reply src/app/components/button/button.component.scss]: Done

File: src/app/app.routes.ts (hunk 4 above)
Comment (a1exymoroz) [reply src/app/app.routes.ts]: Done

**Review by robinsonzimmermann — APPROVED** [review]

**Pull request merged**